src/skeleplex/graph/spline.py

```diff
@@ -47,19 +61,32 @@ def eval(
         derivative : int
             The order of the derivative to evaluate.
             Default value is 0.
+        approx : bool
+            If True, use a quick conversion from normalized arc length
+            coordinates to spline parameter coordinates. If False,
```

Comment:
Can you add a note about the conditions in which the approximation is valid? I believe it is when the knots are equally spaced (arc length).